src/pages/patterns/status-indicator-pattern/index.mdx

```diff
@@ -156,263 +232,157 @@ users, possibly hurting their overall experience.
 </Column>
 </Row>
 
-#### Outline versus filled shapes
-
-We offer two options that can be flexible for usage based on your team's
-preference. Keep in mind that filled icons are more visible and tend to carry
-more weight, therefore they’re often used for high attention statuses. Outlined
-icons are more delicate and not as readily scannable.
-
-In addition, the more line work an icon has—and the more breaks in those
-lines—the more difficult it is to use a filled icon. For this reason, some
-product teams may occasionally mix in an outline icon with a filled icon. This
-is okay but for the most part, designers should try to be consistent throughout
-the product or application. At the very least, designers should avoid mixing
-outlined and filled indicators on the same page.
-
-#### Alias status icons
-
-In several cases, we offer multiple shape options for one type of status
-indicator. For example, ‘warning’ ‘help’ and ‘information’ status icons have
-multiple variants to convey similar or exactly the same information. Often
-times, users of certain products have grown accustomed to say, a hexagon instead
-of a circle to convey a critical warning. Or a team perhaps wants to go the
-extra mile and offer yet another differentiator for accessibility. Although
-consistency is always the goal, there’s no need to avoid differentiation to
-accomodate user preferences. Whichever style you choose however, avoid mixing
-throughout the UI.
-
-If an alias that your product frequently uses has been removed from the set and
-you can make a case for the importance of including it here, please let us know.
-
-### Status palette
+#### Status color palette
 
 The status palette includes all of the colors that can be used to reflect
 status. Typically, red represents danger or error, orange represents a serious
 warning, yellow represents a regular warning, green represents normal or
 success, and blue represents passive notifications, usually involving additional
-information and workflow progress. We’ve also included gray and purple to add
-more depth to the system. Gray indicates drafts or jobs that haven’t been
-started, and purple indicates outliers or undefined statuses.
+information and workflow progress. This status color palette also includes gray
+and purple to add more depth to the system. Gray indicates drafts or jobs that
+haven’t been started, and purple indicates outliers or undefined statuses.
 
 <ColorPalette twoColumn type="status" />
 
 #### Extended status palettes
 
 This palette is only for added contrast accessibility when using yellows and
```

Comment:
I think we should still say these colors are not part of the _brand_ palette and should not be used in other contexts.